python/dgl/graphbolt/impl/neighbor_sampler.py

```diff
@@ -1,10 +1,13 @@
 """Neighbor subgraph samplers for GraphBolt."""
 
+from torch.utils.data import functional_datapipe
+
 from ..subgraph_sampler import SubgraphSampler
 from ..utils import unique_and_compact_node_pairs
 from .sampled_subgraph_impl import SampledSubgraphImpl
 
 
+@functional_datapipe("sample_neighbor")
```

Comment:
ditto.

Reply:
and others, no need to invent new naming convention.

Reply:
I am following the naming style of `datapipe`: https://pytorch.org/data/beta/dp_tutorial.html#naming. then will be consistent.

Reply:
from my perspective, verb looks more reasonable for functional calls.

Reply:
I see, if there is already a name convention, let's follow it.